test/stress/pool-create-hanging.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Test case for figuring out robust way to recognize if connection is dead
+ * for mysql based drivers.
+ */
+const toxiproxy = require('toxiproxy-node-client')
+const toxicli = new toxiproxy.Toxiproxy('http://localhost:8474')
+const rp = require('request-promise-native')
+
+const mssql = require('../../' + process.env.TEST_DRIVER)
+const debug = require('debug')('mssql-stress')
+
+function delay (ms) {
+  return new Promise((resolve, reject) => {
+    setTimeout(resolve, ms)
+  })
+}
+
+function cloneDeep (val) {
```

Comment:
since this is reused, maybe makes sense to extract to test util file? or import lodash as devdependency and use _.cloneDeep?

Reply:
I didn't want to add even extra `dev` dependency for simple one liner, I'll refactor this and delay functions to test utils.